app/assets/stylesheets/style.scss

```diff
@@ -0,0 +1,135 @@
+/*font-family: 'Courgette', cursive;
+font-family: 'Amaranth', sans-serif;
+font-family: 'Nothing You Could Do', cursive;
+font-family: 'Berkshire Swash', cursive;*/
+
+// General Styling   //
+body {
+  
+}
+
+.background-image {
+  position: fixed;
+  left: 0;
+  right: 0;
+  z-index: -1;
+
+  display: block;
+  background-image: image-url("HD-Wallpaper-Cat-With-Beer.jpeg");
+  background-color: #cccccc;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+  width: 100%;
+  height: 100%;
+}
+
+// Home page styling //
+
+.font-pretty {
+  font-family: 'Berkshire Swash', cursive;
+  color: #D68533;
+  font-size: 450%;
+
+  &.smaller {
+    font-size: 215%;
+    padding-left: 2rem;
+  }
+
+}
+
+.recipe-maker-button {
+  
+  .button {
+    background-color: #A32900;
+    margin: 3rem;
+  }
+
+  .button:hover {
+    background-color: #621900;
+  }
+}
+
+.login-to-use {
+  
+  .button {
+    background-color: #A32900;
+    margin: 3rem;
+  }
+
+  .button:hover {
+    background-color: #621900;
+  }
+}
+
+.not-home-page {
+  background-color: white;
+  border-style: solid;
+  border-width: 2px;
+  margin-top: 3rem;
+  padding: 3rem;
+
+  h3 {
+    
+  }
+
+  table {
+   
+    align:center;
+  }
+}
+
+// Recipe List Styling //
+
+.maker-background {
+  position: fixed;
+  left: 0;
+  right: 0;
+  z-index: -1;
+
+  display: block;
+  background-image: image-url("beer-background.jpg");
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+  width: 100%;
+  height: 100%;
+}
+
+#grain-tab a {
+  background-color: #EBC633;
+}
+
+#grain-tab a:hover {
+  background-color: #E6B800;
+}
+
+#hop-tab a {
+  background-color: #5D8D5D;
```

Comment:
Color literals like `#5D8D5D` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#5D8D5D` should be written as `#5d8d5d`